Examples/ServiceExamples/Scripts/ExampleSpeechToText.cs

```diff
@@ -506,9 +684,45 @@ private void OnCheckCustomizationStatus(Customization customization, string cust
         }
     }
 
+    private IEnumerator CheckAcousticCustomizationStatus(string customizationID, float delay = 0.1f)
+    {
+        Log.Debug("TestSpeechToText", "Checking acoustic customization status in {0} seconds...", delay.ToString());
+        yield return new WaitForSeconds(delay);
+
+        //	passing customizationID in custom data
+        _speechToText.GetCustomAcousticModel(OnCheckAcousticCustomizationStatus, customizationID, customizationID);
+    }
+
+    private void OnCheckAcousticCustomizationStatus(AcousticCustomization acousticCustomization, string customData)
+    {
+        if (acousticCustomization != null)
+        {
+            Log.Debug("TestSpeechToText", "Acoustic customization status: {0}", acousticCustomization.status);
+            if (acousticCustomization.status != "ready" && acousticCustomization.status != "available")
+                Runnable.Run(CheckAcousticCustomizationStatus(customData, 5f));
```

Comment:
What does '5f' mean in here ?

Reply:
Yeah I'm torn between keeping and getting rid of the custom logger. I like it doesn't put a bunch of junk in the console and it supports string.Format already. Thats already a time saver. 

5f is number of seconds between status checks. Prob should be 5.0f but I guess Unity didn't complain